spec/spec_helper.rb

```diff
@@ -1,17 +1,15 @@
-dir = File.expand_path(File.dirname(__FILE__))
-$LOAD_PATH.unshift File.join(dir, 'lib')
+require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet-facts'
+include RspecPuppetFacts
 
-require 'mocha'
```

Comment:
I've no clue. This module never got synced and we own it since a very very long time :(